massdash/ui/ConformerPickerUISettings.py

```diff
@@ -39,7 +39,7 @@ def create_ui(self, plot_settings: ChromatogramPlotUISettings):
         plot_settings : ChromatogramPlotUISettings
             The plot settings for the chromatogram.
         """
-        self.shipped_model = st.sidebar.checkbox("Use shipped model", value=True, help="Use the shipped model.")
+        self.shipped_model = st.sidebar.checkbox("Use shipped model", value=True, help="Use the shipped model which picks peaks across 175 points")
         if  self.shipped_model:
             self.pretrained_model_file = os.path.join(DIRNAME, '..', 'assets', 'models', 'conformer', 'base_cape.onnx')
             # Check if the model file exists
```

Comment:
I added a constants.py file recently, can you move the `url_pretrained_conformer` to the constants.py file and then import here.